xclim/indicators/atmos/_conversion.py

```diff
@@ -215,3 +218,28 @@ class Converter(Indicator):
     ),
     compute=wrapped_partial(indices.wind_chill_index, mask_invalid=True),
 )
+
+
+potential_evapotranspiration = Converter(
+    identifier="potential_evapotranspiration",
+    var_name="water_evaporation_flux",
+    units="mm day-1",
+    standard_name="water_evapotranspiration_flux",
+    long_name="Potential evapotranspiration",
+    description=(
+        "The potential for water evaporation from soil and transpiration by plants if the water "
+        "supply is sufficient, with the method {method}."
+    ),
+    compute=indices.potential_evapotranspiration,
+)
+
+water_budget = Converter(
+    identifier="water_budget",
+    units="mm day-1",
```

Comment:
```suggestion
    units="kg m-2 s-1",
```
Same units as both inputs.